repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/list-item.ftl

```diff
@@ -58,6 +58,30 @@
     </#list>
 </#macro>
 
+<#macro newsListItem items categoriesMap>
+    <@hst.link var="pageNotFoundURL" siteMapItemRefId="pagenotfound"/>
+
+    <#list items as item>
+        <@hst.link hippobean=item var="pageURL"/>
+
+        <#if pageURL != pageNotFoundURL>
+            <li>
+                <span class="app-search-results-category">${item.categories?map(category -> categoriesMap[category]!)?join(', ')}</span>
+                <h3><a href="${pageURL}">${item.title}</a></h3>
+                <p class="nhsuk-body-s nhsuk-u-margin-top-1">${item.summary!}</p>
+                <div class="nhsuk-review-date">
+                    <p class="nhsuk-body-s">
+                        <@fmt.message key="published_on.text"/> ${item.publicationDate.time?string['dd MMMM yyyy']}
+                    </p>
+                    <p class="nhsuk-body-s">
```

Comment:
Could we wrap this with `<#if item.author?has_content>...</#if>` as `author` is optional field ?

Reply:
Done. Also added a check on the news page itself. Let me know if I need to change to something mroe elegant.

Reply:
Perfect, thanks 👍